setup.cfg

```diff
@@ -6,3 +6,25 @@ test=pytest
 
 [tool:pytest]
 addopts = --ignore=scripts
+
+[flake8]
+ignore = E501, W503
+select = B,C,E,W,T4,B9,F
+docstring-convention = numpy
```

Comment:
I think you should add the `D` flag to select to also have flake8 do doscstrings